src/main/resources/templates/challenge.html

```diff
@@ -15,12 +15,185 @@
 		<a href="/dashboard" class="primary"><i class="fa fa-graduation-cap"></i>Submit</a>
 	</div>
 </div>
-<div class="challenge-tri">
-<div><textarea class="challenge-input" id="challenge-input">
-// Challenge: Type something that triggers the pretty print.
-// Try something like "var a = document.getElementById('hi');"
-</textarea></div>
-	<div><pre><code id="challenge-pretty" class="js"></code></pre></div>
+<div class="challenge-dual-pane">
+	<div class="left">
+		<div class="challenge-side-header">
+			<div class="title">Simple Dependency Injection</div>
+			<div class="subtitle">Java Challenge</div>
+		</div>
+		<div class="challenge-side-body">
+			<div class="block">
+				Implement a simple dependency injection library that has a single
+				<code>#provide</code> method. You may assume the following:
+			</div>
+			<ul class="block">
+				<li>All injectable constructors are annotated with <code>@Inject</code>.</li>
+				<li>A class named <code>InjectException</code> exists and extends <code>Exception</code>.</li>
+				<li>Only the core Java JDK is allowed. All versions are allowed.</li>
+			</ul>
+			<div class="block">
+				Any other non-listed assumptions must be handled appropriately.
+			</div>
+		</div>
+	</div>
+	<div class="right">
+		<div>
+			<textarea class="code-highlight" id="challenge-input">import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
```

Comment:
Not sure how to request a change for this, but when looking at the front end, if I try interacting with the code, a bunch of the code in the challenge shows up with red underline, even though the package imports look correct. Would you happen to know how to fix that?